charts/pxc-operator/README.md

```diff
@@ -1,23 +1,49 @@
+## Deployment Details
+* <https://kubernetes.io/docs/concepts/workloads/controllers/deployment/>
 
-```bash
-$ helm install . --set watchNamespace=my-namespace
-```
+## Chart Details
+This chart will:
+* deploy a PXC Operator Pod for the further MySQL XtraDB Cluster creation in K8S.
 
-If you need to install the Operator more than once you should disable CRD creation like so:
+### Installing the Chart
+To install the chart with the `pxc` release name using a dedicated namespace (recommended):
 
-```bash
-$ helm install . --set createCRD=false
+```sh
+helm repo add percona https://percona-lab.github.io/percona-helm-charts/
+helm install my-operator percona/pxc-operator --version 0.1.9 --namespace my-namespace
 ```
 
-See [values.yaml](./values.yaml) for a more comprehensive list of settings that can be used to customize the configuration.
+The chart can be customized using the following configurable parameters:
+
+| Parameter                       | Description                                                                   | Default                                   |
+| ------------------------------- | ------------------------------------------------------------------------------| ------------------------------------------|
+| `image.repository`              | PXC Operator Container image name                                             | `percona/percona-xtradb-cluster-operator` |
+| `image.tag`                     | PXC Operator Container image tag                                              | `1.5.0`                                   |
+| `image.pullPolicy`              | PXC Operator Container pull policy                                            | `Always`                                  |
+| `image.pullSecrets`             | PXC Operator Pod pull secret                                                  | `[]`                                      |
+| `replicaCount`                  | PXC Operator Pod quantity                                                     | `1`                                       |
+| `tolerations`                   | List of node taints to tolerate                                               | `[]`                                      |
+| `resources`                     | Resource requests and limits                                                  | `{}`                                      |
+| `nodeSelector`                  | Labels for Pod assignment                                                     | `{}`                                      |
+| `watchNamespace`                | Set when a different from default namespace is needed to watch                | `""`                                      |
+| `nameOverride`                  | Set if the chart name is not desired to be the operators name                 | `""`                                      |
```

Comment:
```suggestion
```
It looks like this option is not officially supported

Reply:
done